foundation_cms/campaigns/models/petition.py

```diff
@@ -9,7 +9,7 @@
 
 
 @register_snippet
```

Comment:
@adalcima **Kudos** for registering these as snippets. This allows us to use the snippet chooser field on the cms rather than a drop down list. I am a fan of the snippet chooser panel because if we dont have a petition created yet it allows the users to create one, its also a little easier to use than a drop down list: 
<img width="3840" height="1756" alt="image" src="https://github.com/user-attachments/assets/32535b12-eed9-48bd-bd1d-6e8926d2e6f9" />